app/lib/package/backend.dart

```diff
@@ -62,18 +62,23 @@ PackageBackend get packageBackend =>
 /// Represents the backend for the pub site.
 class PackageBackend {
   final DatastoreDB db;
+  final Storage _storage;
 
   /// The Cloud Storage bucket to use for uploaded package content.
   /// The following files are present:
   /// - `packages/$package-$version.tar.gz` (package archive)
   /// - `tmp/$guid` (incoming package archive that was uploaded, but not yet processed)
```

Comment:
Need to update this, right?

Reply:
It may be true for the period when the traffic is migrated. Let's update this after this gets released and becomes stable.